tensorflow_encrypted/protocol/pond.py

```diff
@@ -262,11 +269,15 @@ def helper(v: tf.Tensor):
 
     def define_output(
         self,
+        player: Union[str, Player],
         xs: Union['PondPrivateTensor', List['PondPrivateTensor']],
-        receiver: OutputReceiver,
+        post_fn: Callable,
```

Comment:
could we call it `receiver_fn` instead? think that fits better with standard terminology

Reply:
less important with a stronger type for the callable here, but could still be done